### ISSUE TITLE
Generalize Pytorchclassifier to support loss function on logit

### ISSUE THREAD

**Issue by minhitbk** [IBM/adversarial-robustness-toolbox#106]
Currently, the Pytorch classifier only support loss functions that operate on the softmax output. We need to generalize the Pytorch classifier so that it will also support loss functions that are based on the logit output.

**Comment by ririnicolae** [comment]
@minhitbk Is this the same as issue #75?

**Comment by minhitbk** [comment]
@ririnicolae Thats fine, if you already fix it, then we can skip this issue.

**Comment by ririnicolae** [comment]
Fixed in #119 (on `dev`). The loss function provided by the user is applied directly on the output of the model. If the model has logits as output, the loss will be computed on that.
